code/modules/blob/blob.dm

```diff
@@ -1,128 +1,45 @@
-//I will need to recode parts of this but I am way too tired atm
 /obj/effect/blob
-	name = "blob"
+	name = "pulsating mass"
+	desc = "A pulsating mass of interwoven tendrils."
 	icon = 'icons/mob/npc/blob.dmi'
 	icon_state = "blob"
 	light_range = 3
-	light_color = "#b5ff5b"
-	desc = "Some blob creature thingy"
-	density = 1
-	opacity = 0
-	anchored = 1
+	light_power = 4
+	light_color = BLOB_COLOR_PULS
+	density = TRUE
+	anchored = TRUE
 	mouse_opacity = 2
-	layer = 4
+
+	layer = BLOB_SHIELD_LAYER
 
 	var/maxHealth = 30
 	var/health
-	var/regen_rate = 4
-	var/brute_resist = 4
-	var/laser_resist = 4 // Special resist for laser based weapons - Emitters or handheld energy weaponry. Damage is divided by this and THEN by fire_resist.
-	var/fire_resist = 1
-	var/secondary_core_growth_chance = 10.0 //% chance to grow a secondary blob core instead of whatever was suposed to grown. Secondary cores are considerably weaker, but still nasty.
+	var/regen_rate = 5
+	var/brute_resist = 4.3
+	var/fire_resist = 0.8
+	var/laser_resist = 2	// Special resist for laser based weapons - Emitters or handheld energy weaponry. Damage is divided by this and THEN by fire_resist.
 	var/expandType = /obj/effect/blob
-	var/obj/effect/blob/core/parent_core = null
-	var/blob_may_process = 1
-	var/hangry = 0 //if the blob will attack or not.
-	var/blob_cost = 1 //point cost of the blob tile
-
-/obj/effect/blob/New(loc)
-	START_PROCESSING(SScalamity, src)
+	var/secondary_core_growth_chance = 5 //% chance to grow a secondary blob core instead of whatever was suposed to grown. Secondary cores are considerably weaker, but still nasty.
```

Comment:
```suggestion
	var/secondary_core_growth_chance = 5 //% chance to grow a secondary blob core instead of whatever was supposed to grow. Secondary cores are considerably weaker, but still nasty.
```

Reply:
Is this comment even still accurate?